front/server-config/nginx.conf

```diff
@@ -30,4 +30,8 @@ http {
     gzip                on;
 
     include             /etc/nginx/conf.d/*.conf;
+
+    client_body_buffer_size 20m;
```

Comment:
http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size

I think client_body_buffer_size should not be too large.

Reply:
I corrected it, see another pr. This pr will be close because I have created another new branch to handle it.